guides/CODING-GUIDELINES.md

```diff
@@ -30,7 +30,7 @@ JavaScript coding guidelines are enforced using [Nava's ESLint ruleset](https://
 - Build components that are safe to render server-side. This means components must be pure functions of their `props` and `state`, and you only have access to a subset of lifecycle methods:
   - `getInitialState()`
   - `getDefaultProps()`
-  - `componentWillMount()`
```

Comment:
Did one of our tools catch this, or did you? There are probably some other things in here we could update.